Library/OcBootManagementLib/ImageLoader.c

```diff
@@ -940,6 +944,13 @@ InternalEfiStartImage (
   OC_LOADED_IMAGE_PROTOCOL   *OcLoadedImage;
   EFI_LOADED_IMAGE_PROTOCOL  *LoadedImage;
 
+  if (  (mImageLoaderConfigure  != NULL)
+     && (mImageLoaderCapsHandle != ImageHandle))
+  {
+    ASSERT (mImageLoaderCapsHandle == ImageHandle);
```

Comment:
If you really want an `ASSERT`, make it `ASSERT(FALSE)`, but why not just use a print statement which can also output the handles?